tests/installation/playwright-should-work.spec.ts

```diff
@@ -15,7 +15,7 @@
  */
 import { test, expect } from './npmTest';
 
-test(`playwright should work`, async ({ exec, nodeMajorVersion, installedSoftwareOnDisk }) => {
```

Comment:
shall we use realCDN here to have one full simple e2e test that executes main workflow?